src/Mono.Android/Test/Mono.Android-Tests.csproj

```diff
@@ -18,6 +18,7 @@
     <AndroidManifest>Properties\AndroidManifest.xml</AndroidManifest>
     <AndroidSupportedAbis>armeabi-v7a;x86</AndroidSupportedAbis>
     <AndroidUseSharedRuntime>False</AndroidUseSharedRuntime>
+    <AndroidEnableSGenConcurrent>true</AndroidEnableSGenConcurrent>
```

Comment:
Should we consider making this the default for all projects?

We already do it for .NET 6:

https://github.com/xamarin/xamarin-android/blob/885b57bdcf32e559961b183e1537844c5aa8143e/src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.DefaultProperties.props#L13

Reply:
Good question, I remember us wanting to do this in the past but I think decided to only do so in the templates. Maybe enough time has passed since that change for us to want this everywhere, or maybe we shouldn't enable this for all existing projects. @jonpryor @steveisok do either of you have an opinion on this?